packages/forms-web-app/src/dynamic-forms/questions.js

```diff
@@ -285,7 +285,7 @@ exports.questions = {
 		title: 'Appeals near the site',
 		question: 'Are there any other ongoing appeals next to, or close to the site?',
 		pageTitle: 'Are there any other ongoing appeals near the site?',
-		fieldName: 'other-ongoing-appeals',
+		fieldName: 'ongoing-appeals',
```

Comment:
maybe put in an url field with the new url, and preserve the existing fieldName?